WTWSMS/common/nicknames/00_nicknames.txt

```diff
@@ -452,7 +452,8 @@ nick_the_fair = {
 		NOT = { age = 50 }
 	}
 	chance = {
-		factor = 1
+		factor = 1.5
+
```

Comment:
That was intentional. A person with extremely good looks is likely to be remembered by it.